### ISSUE TITLE
Temperature  limiter is semi-broken

### ISSUE THREAD

**Issue by Sidepie** [AppHouseKitchen/AlDente-Charge-Limiter#1332]
**MacBook (please complete the following information):**
 - M1Pro
 - Sequoia
 - Apple 120W
 - USB-C, Apple standard
 - AlDente Pro
 - AlDente Version: 1.28.3
 - External Monitor connected? - No
 - MacBook used in Clamshell Mode (with the lid closed)? - No

**Describe the bug**
On my machine, charging stops when the temperature exceeds 35°C. 
However, once charging stops, it does not restart, even if the temperature falls back below the threshold and more than 5 minutes have passed. 
In other words, the hysteresis process described in the documentation does not occur, except for stopping when the temperature limit is initially reached.

**Expected Behavior**
The charging temperature limit should work as described

**Additional context**
If I manually move the charging limit, up or down, in the popup, with the mouse, the charging is instantly resumed.

**Comment by MatthiasKerbl** [comment]
Hi @Sidepie,

If possible, please share a debug file the next time this happens so that we can assess this issue better. You can find a guide on [How to generate and share a debug file](https://apphousekitchen.com/how-to-generate-and-share-a-debug-file/) on our blog.

Best,
Matthias

**Comment by jkittikun** [comment]
I got the same problem. Charging does not continue after heat protection triggered.

- MacBook Air M1
- Sequoia
- Apple 30W
- USB-C, Apple standard
- AlDente Pro
- AlDente Version: 1.28.3

<img width="200" alt="Screenshot 2024-09-20 at 08 29 48 2" src="https://github.com/user-attachments/assets/0a1add23-258b-4178-ac7f-77e350e4c8fa">

**Comment by MatthiasKerbl** [comment]
Hi everyone,

We have just released AlDente 1.28.4 which includes a fix for this issue. To get AlDente 1.28.4 you need to update within the app under Settings/General/Check for Updates.
Please report back if it works for you.

Best,
Matthias

**Comment by Sidepie** [comment]
Hi,
Thank you for your fast support, everything seems to be in order now.
